infrastructure/builder/src/generate/generators/monitoring.js

```diff
@@ -0,0 +1,69 @@
+const path = require('path');
+const cronstrue = require('cronstrue');
+const table = require('markdown-table');
+const {listServices, writeRepoFile} = require('../../utils');
+
+const SERVICES = listServices();
+
+exports.tasks = [];
+
+SERVICES.forEach(name => {
+  exports.tasks.push({
+    title: `Generate Monitoring Hints for ${name}`,
+    requires: [
+      `procs-${name}`,
+    ],
+    provides: [`monitoring-hints-${name}`],
+    run: async (requirements, utils) => {
+      const res = [];
+      Object.entries(requirements[`procs-${name}`]).forEach(([proc, ext]) => {
+        if (ext.type === 'cron') {
+          res.push([name, proc, `taskcluster.${name}`, ext.deadline, cronstrue.toString(ext.schedule)]);
+        }
+        if (ext.type === 'background' && ext.subType === 'iterate') {
```

Comment:
What about handler processes?

Reply:
How would one monitor that? They could potentially never have an event fire and it would be fine.

Reply:
Good point.  We do automatically reconnect, so there's potential to say something here, but let's leave it for another PR in the indefinite future :)